src/main/java/org/broadinstitute/hellbender/tools/exome/ReadCountCollectionUtils.java

```diff
@@ -614,7 +661,49 @@ public static ReadCountCollection removeTargetsWithTooManyZeros(final ReadCountC
      *   is no target to be dropped.
      */
     public static ReadCountCollection removeTotallyUncoveredTargets(final ReadCountCollection readCounts, final Logger logger) {
-        return removeTargetsWithTooManyZeros(readCounts, readCounts.columnNames().size() - 1, logger);
+        return removeTargetsWithTooManyZeros(readCounts, readCounts.columnNames().size() - 1, true, logger);
+    }
+
+    /**
+     * Remove targets that have read counts strictly lower than {@code minReadCount} for all
+     * samples.
+     * <p>
+     *     It will return a copy of the input read-count collection with such targets dropped.
+     * </p>
+     *
+     * TODO github/gatk-protected issue #843 -- write unit test
+     *
+     * @param readCounts the input read counts.
+     * @param minCoverage minimum read count
+     * @return never {@code null}. It might be a reference to the input read-counts if there is
+     *   is no target to be dropped.
+     */
+    public static ReadCountCollection removeTargetsWithUniformlyLowCoverage(final ReadCountCollection readCounts,
+                                                                            final int minCoverage,
+                                                                            final Logger logger) {
+        /* filter targets to keep, put them in an ordered set */
```

Comment:
I think there are arguments for using `//`, but the most relevant argument is simply for consistency of style in the codebase!